src/data/int/parity.lean

```diff
@@ -220,6 +220,24 @@ begin
     norm_num },
 end
 
+lemma even.two_mul_div_two : even n → 2 * (n / 2) = n :=
+by { rintro ⟨c, rfl⟩, simp }
```

Comment:
It makes no sense to put these lemmas into `even`/`odd` namespaces because we have a global definition `even`, not namespaced definitions `nat.even` and `int.even`. Also, this lemma is definitionally equal to
```suggestion
lemma two_mul_div_two (h : even n) : 2 * (n / 2) = n := nat.mul_div_cancel_left' h
```

Reply:
@urkud I agree that it doesn't make much sense, but there are a number of other lemmas in this file that do the same thing so they ought to be changed, too, at some point.

Do you think `two_mul_div_two_of_even` would be better here? There's likely going to be `two_mul_div_two_of_odd` as well.

Reply:
@urkud So, you suggest to remove `even`/`odd` from the name completely?

Reply:
`_of_even` and `_of_odd` seem more suitable.